.gitignore

```diff
@@ -18,3 +18,5 @@ node_modules
 .build
 yarn.build.json
 ssh-keypair*
+
+scripts/out
```

Comment:
Might be good to delete the git keep file for this

Reply:
Did I mess this up? I wanted to keep the directory but ignore it's contents